### PR TITLE
Change Besu repository and chat link

### PR DESCRIPTION
**Pull request by timbeiko**
Pantheon is now called Hyperledger Besu and has changed repo and chat platform.

### CODE REVIEW

File: docs/developers/index.md [hunk 1]
```diff
@@ -252,10 +252,10 @@ The Ethereum network is made up of many nodes who run compatible client software
 - [parity.io](https://www.parity.io/)
 - [Github](https://github.com/paritytech/parity-ethereum)
 
-### Besu (Pantheon) *Ethereum client written in Java*
+### Hyperledger Besu (Formely Pantheon) *Ethereum client written in Java*
 - [pegasys.tech](http://pegasys.tech)
-- [Github](https://github.com/PegaSysEng/pantheon/)
-- [Gitter](https://gitter.im/PegaSysEng/pantheon)
+- [Github](https://github.com/hyperledger/besu)
+- [Rocket.Chat](https://chat.hyperledger.org/channel/besu)
```
Comment (samajammin) [inline docs/developers/index.md]: Can you please confirm this link is correct? This URL hangs for me when I try to access it.

Comment (timbeiko) [reply docs/developers/index.md]: @samajammin thanks for checking! This is indeed the right link. You need to be logged in to see the conversation. We've already raised an issue with Rocket.Chat about this: https://github.com/RocketChat/Rocket.Chat/issues/15391

Comment (samajammin) [reply docs/developers/index.md]: Bummer. Definitely a suboptimal UX. Any thoughts on how we can improve/avoid that for users? Perhaps link to the chat home (https://chat.hyperledger.org/) instead & mention the #besu channel? Or wait until that issue is resolved?

**Comment by samajammin** [comment]
@timbeiko never heard back from you but it looks like this was solved in #370 so I'll close this out. Future contributions always welcome! Thanks.

**Pull request closed without merging**